draft-ietf-quic-qpack.md

```diff
@@ -937,11 +939,11 @@ entries, the Base will be greater than the Required Insert Count, so the delta
 will be positive and the sign bit is set to 0.
 
 An encoder that produces table updates before encoding a header block might set
-Required Insert Count and the Base to the same value.  In such case, both the
```

Comment:
Rewording to emphasize that the encoder has very limited freedom in what to set Required Insert Count to.